rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -1790,7 +1792,8 @@ void opcode_009a(const ScriptArguments& args, const ScriptPedType pedType, const
     RW_UNUSED(pedType);
 
     coord = script::getGround(args, coord);
-    character = args.getWorld()->createPedestrian(model, coord + script::kSpawnOffset);
+    character = args.getWorld()->createPedestrian(model, coord);
```

Comment:
Yes, before returning the object an offset can be applied. But do we want do do this?
The original game also sets the center position of objects and applies an offset when creating objects via opcode.
It would make sense when `createPedestrian|Vehicle|Player` only gets called with coordinates referring to the ground, is this/will this be the case?